pkg/relayer/protocol/block_heights.go

```diff
@@ -0,0 +1,37 @@
+package protocol
+
+import (
+	"encoding/binary"
+	"log"
+	"math/rand"
+
+	"github.com/pokt-network/poktroll/pkg/client"
+)
+
+func GetCreateClaimDistributionHeight(earliestCreateClaimBlock client.Block) int64 {
```

Comment:
Add a comment that we need to add tests for these `TODO_TEST(@bryanchriswhite): ...`